Cargo.toml

```diff
@@ -61,17 +57,52 @@ winit = { version = "0.27.1", default-features = false, features = ["wayland", "
 x11rb = { version = "0.10.0", optional = true }
 xkbcommon = "0.4.0"
 scan_fmt = { version = "0.2.3", default-features = false }
+wayland-egl = {  path = "../wayland-rs/wayland-egl", optional = true }
+wayland-protocols = {  path = "../wayland-rs/wayland-protocols", features = ["unstable", "staging", "server"], optional = true }
+wayland-protocols-wlr = {   path = "../wayland-rs/wayland-protocols-wlr", features = ["server"]}
+wayland-server = {   path = "../wayland-rs/wayland-server",  optional = true }
+wayland-sys = {  path = "../wayland-rs/wayland-sys",  optional = true }
+wayland-backend = {  path = "../wayland-rs/wayland-backend", optional = true }
+winit = { version = "0.26", optional = true }
+x11rb = { version = "0.9.0", optional = true }
+xkbcommon = "0.4.0"
+scan_fmt = { version = "0.2.3", default-features = false }
+
+# This dependency will only be included when targeting Android
+[target.'cfg(target_os = "android")'.dependencies]
+ndk-glue = "0.6.2" # Substitute this with the latest ndk-glue version you wish to use
+
+
+ndk = "0.5.0"
+
+[target.'cfg(not(target_os = "android"))'.dependencies]
+
+udev = { version = "0.6", optional = true }
+libseat= { version = "0.1.1", optional = true }
+input = { version = "0.7", default-features = false, features=["libinput_1_19"], optional = true }
+drm-fourcc = "^2.1.1"
+drm = { version = "0.6.1", optional = true }
+drm-ffi = { version = "0.2.1", optional = true }
+gbm = { version = "0.8.0", optional = true, default-features = false, features = ["drm-support"] }
+input = { version = "0.7", default-features = false, features =["libinput_1_14"], optional = true }
```

Comment:
Yeah this is probably an iron clad reason to work on the cratening for smithay.

Reply:
I think it's fine to move ahead?